js/mexc3.js

```diff
@@ -4032,6 +4033,68 @@ module.exports = class mexc3 extends Exchange {
         };
     }
 
+    async borrowMargin (code, amount, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name mexc3#borrowMargin
+         * @description create a loan to borrow margin
+         * @see https://mxcdevelop.github.io/apidocs/spot_v3_en/#loan
+         * @param {string} code unified currency code of the currency to borrow
+         * @param {float} amount the amount to borrow
+         * @param {string} symbol unified market symbol
+         * @param {object} params extra parameters specific to the mexc3 api endpoint
+         * @param {string|undefined} params.marginMode 'cross' or 'isolated' only 'isolated' is supported for borrowMargin()
+         * @returns {object} a [margin loan structure]{@link https://docs.ccxt.com/en/latest/manual.html#margin-loan-structure}
+         */
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        const currency = this.currency (code);
+        const request = {
+            'asset': currency['id'],
+            'amount': this.currencyToPrecision (code, amount),
+        };
+        const [ marginMode, query ] = this.handleMarginModeAndParams ('borrowMargin', params);
+        if (marginMode === 'cross') {
```

Comment:
I think we can simplify this code a bit, since we only support `isolated` and isolated requires a symbol, symbol will always be mandatory so we can move the `symbol === undefined` to the beginning of the method and we can avoid adding unnecessary if/else nesting.

Example:

```Javascript
if (symbol === undefined) {
                throw new ArgumentsRequired (this.id + ' borrowMargin() requires a symbol argument.');
 }
await this.loadMarkets ();
(...)
const request = {
            'asset': currency['id'],
            'amount': this.currencyToPrecision (code, amount),
             'symbol': market['id']
        };
(...)
```